src/components/Switch/Switch.types.ts

```diff
@@ -0,0 +1,40 @@
+import { SwitchClasses } from './Switch.classes';
+
+import { SxProps, Theme } from '@mui/material/styles';
+
+export interface SwitchProps {
+  className?: string;
+  /** Override or extend the styles applied to the component. */
+  classes?: Partial<SwitchClasses>;
+  /** The system prop that allows defining system overrides as well as additional CSS styles. */
+  sx?: SxProps<Theme>;
+  /** If `true`, the component is checked. */
+  checked?: boolean;
+  /** Callback fired when the state is changed. */
+  onChange?: (value: React.ChangeEvent<HTMLInputElement>) => void;
+
+  /**
+   * The color of the component.
+   * @default 'primary'
+   */
+  color?: 'primary' | 'secondary' | 'error' | 'info' | 'success' | 'warning';
+
+  /** If 'button', hitting enter will toggle the switch. */
+  type?: 'checkbox' | 'button';
+  /** Set component size */
```

Comment:
```suggestion
  /** The size of the component. */
```
И указать `@default`